src/telemetry/sdk/remote.ts

```diff
@@ -0,0 +1,108 @@
+/**
+* This is an auto generated code. This code should not be modified since the file can be overwriten 
+* if new genezio commands are executed.
+*/
+
+let http: any = null;
+let https: any = null;
+let importDone = false;
+
+async function importModules() {
+    if (typeof process !== "undefined" && process.versions != null && process.versions.node != null) {
+        const httpModule = 'http';
+        http = await import(httpModule);
+        const httpsModule = 'https';
+        https = await import(httpsModule);
+    }
+    importDone = true;
+}
+
+async function makeRequestBrowser(request: any, url: any) {
```

Comment:
```suggestion
async function makeRequestBrowser(request: any, url: string) {
```
Shouldn't this be a string?

Reply:
this is coming from genezio sdk

Reply:
We should remove these anys and put actual types. Let's make a task with all the improvements suggested by Costin.